M2/Macaulay2/packages/NumericalAlgebraicGeometry/extraNAGtypes.m2

```diff
@@ -97,6 +97,47 @@ toExternalString fS
 toExternalString fT
 ///
 
+--todo: bring into harmony with (jacobian, PolySystem)
```

Comment:
@antonleykin  ---   are these changes to your package acceptable?

Reply:
> @antonleykin --- are these changes to your package acceptable?

Yes, Tim made these on my request.